docs/metadata.md

```diff
@@ -0,0 +1,49 @@
+Adding business context to a system using metadata
+==================================================
+
+Adding business context to Sigrid makes it easier to interpret the results. For example, a below-market average maintainability rating of 2 stars might seem like a problem, but this depends entirely on the context. If the system is no longer actively maintained and will be decommissioned by the end of the year, such a rating might be perfectly acceptable. But if that system is new, uses a modern technology, and is business critical, such a rating would be considered a red flag. In both cases the technical conclusion is identical, it's the context that determines the urgency.
+
+This context information is called *metadata* in Sigrid. Adding metadata can be done in 3 different ways:
+
+### Option 1: Adding metadata in Sigrid
+
+This is by far the simplest approach: in Sigrid, simply go to the system settings option in the menu and select the "metadata" option. 
+
+<img src="images/metadata-ui.png" width="600" />
+
+### Option 2: Using the Sigrid API to add metadata
+
+The [Sigrid API end point for metadata](sigrid-api-documentation.md#system-metadata) allows you to add metadata programmatically. This is less accessible than using the Sigrid user interface, but has the advantage that it can run automatically. This is typically used when you need to synchronize metadata from another system to Sigrid, and you want this to run in an automated way.
+
+### Option 3: Adding metadata from a YAML file in your repository
+
+As an alternative to using the API, you can also create a YAML file called `sigrid-metadata.yaml` in the root of your repository. The contents of this file are then used to update the metadata when the analysis runs. This is similar to using the API, but allows you to manage your Sigrid metadata as part of the repository in your version control system.
+
+The following examples shows an example of a `sigrid-metadata.yaml` file:
+
+```
+metadata:
+  divisionName: My division
+  supplierNames:
+    - Team 1
+    - Team 2
+  lifecyclePhase: "EOL",
```

Comment:
Double quotes are not needed in Yaml (and you do not have them in `divisionName`), the commas are either syntax errors, or will be appended to the string (I think). I suggest we remove both for clarity.

Reply:
The quotes and commas are both a result of me find-replacing the JSON into YAML :) They do recommend using quote for free-format text, see https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml, so I left them on the remark field and removed them for the fields with simple values.